docs/docs/general/07-Troubleshooting/index.md

```diff
@@ -3,3 +3,11 @@
 If you are having problems using the askui lib, have a look at the following pages. If you can't solve your
 challenge after you have read through the pages, please drop us a note. We are happy to help!
 
+## Pages
+* [ipv6](ipv6.md)
+* [linux](linux.md)
+* [mac-os](mac-os.md)
+
+# Support
+
+* [Discord Community](https://discord.gg/UHSZ99Xhx2)
```

Comment:
Like :) Maybe, we can make it even more explicit of how to join our community on Discord (the individual steps from registration/login to writing us) so that people know how much effort this is, especially, people not yet using Discord.

Reply:
Added a sentence that you have to create an account on Discord if you do not have one already.